spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/security/oauth2/resource/OAuth2ResourceServerProperties.java

```diff
@@ -71,6 +72,11 @@ public Opaquetoken getOpaquetoken() {
 		 */
 		private Resource publicKeyLocation;
 
+		/**
+		 * Identifies the recipients that the JWT is intended for.
+		 */
+		private String audience;
```

Comment:
Consider the spring-cloud-gateway scenario, I suggest to Use `List<String>` instead of `String`.
```
client -> spring-cloud-gateway -> resource-server-1
                               -> resource-server-2
                               -> etc..
```
When spring-cloud-gateway validate the access-token, the valid audience is a list of string.

Reply:
And rename variable from `audience` to `audiences`.

Reply:
Thanks, @chenrujun. This makes sense to me. The validation should then use a predicate that checks that at least one audience in the JWT matches one of the configured audiences. Does that makes sense to you, @mbhave?